docs/notebooks/dmri_covariance.ipynb

```diff
@@ -76,6 +85,89 @@
     "plt.xlabel(\"Angular distance\")\n",
     "plt.ylabel(\"Covariance (arbitrary scaling)\")\n",
     "\n",
+    "plt.legend(frameon=False);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e358167b-21ad-4ee7-b85d-b26affb15f46",
+   "metadata": {},
+   "source": [
+    "## Extracting empirical covariances\n",
+    "\n",
+    "Plot empirical covariances of all voxels within the brain mask from one HCP subject."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "7d47dcc8-d7f5-4efe-8124-f8e23a2d2afd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hcpdata = np.load(\"hcpdata.npz\")\n",
```

Comment:
What does `hcpdata.npz` contain? np.cov only accepts 1D or 2D data. Having a look at the `triu_indices` later, I assume it is 2D, so how have you reshaped the data? It would be relevant to add some context in the notebook. Is it light enough to add it to the repository or somewhere where it can be downloaded?

Reply:
The file contains bvalues, bvectors, and a 2D array of data.

The data array is of shape (90, N) where 90 corresponds to b ∈ [2500, 3500] and N is number of brain voxels in the brain mask. We may want to exclude the cerebellum and ventricles in the future (Anderson 2015's fig 3 talks about cerebral voxels)

Reply:
Btw, I didn't add it in the PR because it's 250MB